openshift_cli_installer/libs/managed_clusters/acm_clusters.py

```diff
@@ -29,8 +34,14 @@ def install_acm(
     registry_config_file,
     timeout_watch,
 ):
+    section = "Install ACM"
     cluster_name = hub_cluster_data["name"]
-    click.echo(f"Installing ACM on cluster {cluster_name}")
+    platform = hub_cluster_data["platform"]
+    aws_access_key_id = hub_cluster_data["aws-access-key-id"]
```

Comment:
need to add `hub_cluster_data["platform"] == GCP_OSD_STR:`  here as well

Reply:
I don't understand this comment